### ISSUE TITLE
it doesn't connect.

### ISSUE THREAD

**Issue by WaltDisneyWorld** [matallui/demo-screen-capture#24]
Doesn't link to a target...

**Comment by matallui** [comment]
@WaltDisneyWorld This is a demo from a couple of years ago. It should work as it is. Also, this is not a maintained library or anything, only code for you to use as reference, in case you're trying to achieve the same goal as I was back then.

Are you saying the demo doesn't work anymore? Is that it?

**Comment by WaltDisneyWorld** [comment]
Yes, what I am trying to say is that the Demo needs to link to a target with a TeamID in order for the app to process at All.

**Comment by matallui** [comment]
Oh yeah, you need to have Xcode set up and have a dev account I believe. I guess whatever the requirements would be for you to build a native iOS application, right?

All we could potentially do here is update the readme file with some requirements. You're welcome to submit a PR if you'd like to improve this.
